components/renku_data_services/notebooks/api.spec.yaml

```diff
@@ -723,6 +876,158 @@ components:
       - renku.io/projectName
       - renku.io/repository
       type: object
+    SessionPostRequest:
+      properties:
+        project_id:
+          $ref: "#/components/schemas/Ulid"
+        launcher_id:
+          $ref: "#/components/schemas/Ulid"
+        storage:
```

Comment:
Consider renaming this to disk_storage for better clarity.

Reply:
Done